<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ButtonsMD</class>
 <widget class="QWidget" name="ButtonsMD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>92</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>31</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>92</width>
    <height>10000018</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frm_bt">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>9</y>
     <width>31</width>
     <height>132</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>31</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>31</width>
     <height>1000000</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="bt_edit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {
background-color: rgb(255, 255, 255);
color: rgb(21, 151, 255);
border-color:  rgb(21, 151, 255);
border-style: solid;
border-width: 2px;
}

QPushButton:pressed {
 background-color: rgb(0, 99, 149)
}

QPushButton:disabled {
    background-color: rgb(100, 100, 100)
}

QPushButton{
background-color: rgb(0, 170, 255);
border-radius: 15px;
color: rgb(255, 255, 255);
font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_del">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {
background-color: rgb(255, 255, 255);
color: rgb(21, 151, 255);
border-color:  rgb(21, 151, 255);
border-style: solid;
border-width: 2px;
}

QPushButton:pressed {
 background-color: rgb(0, 99, 149)
}

QPushButton:disabled {
    background-color: rgb(100, 100, 100)
}

QPushButton{
background-color: rgb(0, 170, 255);
border-radius: 15px;
color: rgb(255, 255, 255);
font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_add">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover {
background-color: rgb(255, 255, 255);
color: rgb(21, 151, 255);
border-color:  rgb(21, 151, 255);
border-style: solid;
border-width: 2px;
}

QPushButton:pressed {
 background-color: rgb(0, 99, 149)
}

QPushButton:disabled {
    background-color: rgb(100, 100, 100)
}

QPushButton{
background-color: rgb(0, 170, 255);
border-radius: 15px;
color: rgb(255, 255, 255);
font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
